src/Server.cpp

```diff
@@ -122,13 +123,29 @@ Server& Server::operator=(const Server& ref)
 void Server::run()
 {
 	RequestParser parser;
+	const int maxEvents = 10;
+	const int timeout = 5000; // epoll timeout in milliseconds
+	const int acceptedIdleTime = 10; // time in seconds until an idle connection is closed
 
 	while (true) {
-		struct epoll_event events[MAX_EVENTS];
+		struct epoll_event events[maxEvents];
 		// Blocking call to epoll_wait
-		int nfds = epoll_wait(m_epfd, events, MAX_EVENTS, -1);
+		int nfds = epoll_wait(m_epfd, events, maxEvents, timeout);
 		if (nfds < 0)
 			throw std::runtime_error("epoll_wait");
+		if (nfds == 0) {
+			LOG_WARN << "Timeout: no events received within " << timeout << " ms.";
```

Comment:
In #22 I have it as LOG_DEBUG.
I didn't see it as warning since not getting any (new) events seems "normal" to me for a webserver. We can do stuff like handleTimeout when no event occured.

Reply:
You have it as LOG_INFO even. But is fine by me.

Reply:
Leave as LOG_INFO or switch LOG_DEBUG? Right now LOG_DEBUG makes more sense to me :D